pkg/up/up.go

```diff
@@ -596,10 +596,13 @@ func (u *upRunner) updateAppMaxObservedPodStatus(pod *v1.Pod) error {
 					Follow:    true,
 					Container: containerStatus.Name,
 				}
+				// Capture the pod name in a variable. We cannot use the pod variable in the goroutine because it is
+				// invalid after this function returns.
+				podName := pod.ObjectMeta.Name
```

Comment:
Based on feedback from the linter that Michael is running, a better fix here is to ensure that the pod pointer passed to this function is not a pointer to a range variable, but rather a pointer to the slice element.

Reply:
Jasper fixed this.